uploadNeuroDB/minc_insertion.pl

```diff
@@ -145,43 +145,46 @@
 
 USAGE
 &Getopt::Tabular::SetHelp($Help, $Usage);
-&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
+&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
 
+if (!$ENV{LORIS_CONFIG}) {
+	print STDERR "\n\tERROR: Environment variable 'LORIS_CONFIG' not set\n\n";
+	exit 1;  # TODO replace by appropriate exit code from ExitCodes once it is defined
```

Comment:
why not define it now?

Reply:
The $ENV{'LORIS_CONFIG'} check should be done in *all* scripts if we want to be consistent. I was thinking of issuing a PR that does just that (along with creating a new entry for the exit code in ExitCode.pm). Agreed?

Reply:
sounds good to me. It might be a good idea to just define it here as a variable in the ExitCodes.pm class, and re-use that code in other scripts at a later time
(simply to avoid telling the users at release time that exit 1 again changed meaning)....

Reply:
My next commit covers this.